dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/OwnershipTests.java

```diff
@@ -126,7 +125,7 @@ public void shouldUpdateTrackedEntitiesInSearchScopeWhenGlassIsBroken() throws E
 
     JsonObject updatePayload =
         trackerImportExportActions
-            .getTrackedEntity(teId + "?fields=*")
+            .getTrackedEntity(teId + "?fields=*,!enrollments")
```

Comment:
why do we not need enrollments anymore here? and why did the error codes change?

Reply:
We were getting tracked entities and enrollments and events and update them but user hasn't the event org unit in capture scope and then it is not allowed to update. This specific test is for ownership, so just retrieving and updating tracked entities makes sense.

In the other test, the error code changes because the user doesn't have org unit in capture scope for events and this is now validated also for updates.

@karolinelien and @ameenhere confirmed that this new behavior is correct, as soon as @muilpp have a look I am going to create a bug and see if we need to backport it.